_sections/100-encryption.md

```diff
@@ -0,0 +1,109 @@
+    <meta xmlns="http://openrosa.org/xforms">
+        <instanceID>uuid:5b9cf8d1-106f-4004-844f-c072d76762ed</instanceID>
+    </meta>
+</data>
+```
+
+The following elements are supported. Unless otherwise specified all attributes and elements should be in the `"http://opendatakit.org/submissions"` namespace.
+
+| `<element>`/attribute             | description
+|-----------------------------------|-------------
+|`<data>`                           | The required root element.
+|          `id`                     | Required on the root element with same value as the `id` attribute in the [Primary Instance](/#primary-instance) of the XForm.
+|         `encrypted`               | Required on the root element with the value "yes" indicating this is an encryption manifest.
+|         `version`                 | Required on the root element with same value as the `version` attribute in the [Primary Instance](/#primary-instance) of the XForm but only if it exist there.
+|`<base64EncryptedKey>`             | Required child of the root element with the value of the [encrypted encryption key](#key-encryption) used to encrypt the record.
+|`<encryptedXmlFile>`               | Required child of the root element with the value of the filename of the encrypted XML file.
+|`<media>`                          | Child of the root element, which is required if the record contains any additional media files.
+|`<file>`                           | Child of the media element, which is required for each media file belonging to the record. The order of multiple file elements is [significant](#content-encryption).
+|`<base64EncryptedElementSignature>`| Optional child of the root element that contains a [signature](#signature) that can be used to validate that the encrypted record does not appear to be tampered with.
+|`<meta>`                           | Required child of the root element in the `"http://openrosa.org/xforms"` namespace.
+|`<instanceID>`                     |  Required child of the meta element in the `"http://openrosa.org/xforms"` namespace containing the value of the identical element in the XML record.
+
+### Content Encryption
+
+The XML file and uploaded files are encrypted with the equivalent of the **AES/CFB/PKCS5Padding** algorithm as used in Java 9 with a [specified initialization vector algorithm](#initialization-vector). 
+
+The files should be encrypted in the following order:
+
+1. all media files one by one
+2. the XML file
+
+The order in which the `<file>` elements are recorded in the submission manifest has to match the order in which they were encrypted.
+
+#### AES Encryption Key
+
+The AES encryption is a random 256 bit key generated by the client for each record. The submission manifest contains a base64-encoded string of this key.
+
+After encrypting all files belonging to the record and generating the submission manifest, it is thrown away.
```

Comment:
The "it" that is thrown away is the key, right? I might say "the raw key" or something because my first couple of readings suggested the manifest was thrown away which doesn't make sense.

Reply:
Indeed! Thanks. Will change!